### DISCUSSION TITLE
Allow to create arrows with a single click

### DISCUSSION THREAD

**Discussion by danielo515** [zsviczian/obsidian-excalidraw-plugin#1309] [Ideas]
It will be awesome to be able to link shapes with an arrow using a single click on both ends. Currently doing a single click makes the arrow bend, which is not what I want. I never want to do that, so it is very frustrating that it happens every time

**Comment by Rez-Shok** [comment]
Hi Daniel, there should be some ways for people that want bend there arrows. The sngle click for bend - double click for straight arrow in my opinion is fair enough. You can also add your arrow or drawing or almost everything to your library and use it with single click.
